cmd/talosctl/cmd/talos/etcd.go

```diff
@@ -36,6 +36,20 @@ var etcdLeaveCmd = &cobra.Command{
 	},
 }
 
+var etcdMemberRemoveCmd = &cobra.Command{
+	Use:   "remove-member <hostname>",
+	Short: "Remove the node from etcd cluster",
+	Long:  ``,
```

Comment:
I feel we should add more description here saying that this might destroy etcd cluster if used for a running member, etc.

And say something like the usecase for this command is recovering from the scenario when etcd member is already in a broken state (e.g. hardware failure), and we need to remove that member without having access to the actual node.

Reply:
ok, sure